agent/runner/jobs/pbm_helpers.go

```diff
@@ -533,3 +534,23 @@ func groupPartlyDoneErrors(info describeInfo) error {
 	}
 	return errors.New(strings.Join(errMsgs, "; "))
 }
+
+// pbmGetSnapshotTimestamp returns time the backup restores target db to.
+func pbmGetSnapshotTimestamp(ctx context.Context, dbURL *url.URL, backupName string) (*time.Time, error) {
+	var list pbmList
+	if err := execPBMCommand(ctx, dbURL, &list, "list"); err != nil {
+		return nil, err
+	}
+
+	if len(list.Snapshots) == 0 {
+		return nil, errors.Wrapf(ErrNotFound, "got no one snapshot")
+	}
+
+	for _, snapshot := range list.Snapshots {
+		if snapshot.Name == backupName {
+			return pointer.ToTime(time.Unix(snapshot.RestoreTo, 0)), nil
+		}
+	}
+
+	return nil, errors.Wrapf(ErrNotFound, "couldn't find required snapshot")
```

Comment:
nothing to wrap methinks (same as above).
```suggestion
	return nil, errors.Wrap(ErrNotFound, "couldn't find required snapshot")
```